### ISSUE TITLE
​ Limited customer and invoices search with react framework

### ISSUE THREAD

**Issue by mirzaishfaq** [invoiceninja/ui#1342]
with react framework I cannot search customer and invoices with with phone number.
is there anyway I can edit a code and search like a Flutter framework?

**Comment by turbo124** [comment]
@mirzaishfaq 

yes this is known, the react application didn't like a large amount of data in that searchable section, so we had to trim the static file down to only a subset of data. we are looking into implementing a live search which will cover this issue.

**Comment by mirzaishfaq** [comment]
any idea how long will it take to fix the search issue with react application. I think it should be easy fix. Just need to add variable in search filter. We just installed invoice ninja with Flutter app and sadly it  has a slow response and it is taking too much CPU usage.

**Comment by mirzaishfaq** [comment]
@turbo124

any idea how long will it take to fix the search issue with react application. I think it should be easy fix. Just need to add variable in search filter. We just installed invoice ninja with Flutter app and sadly it has a slow response and it is taking too much CPU usage.

**Comment by turbo124** [comment]
we have improved the global search. please reopen if you have any additional issues with this.
